cfgov/cfgov/urls.py

```diff
@@ -136,17 +137,6 @@
             name='index')],
         namespace='contact-us')),
 
-    url(r'^events/', include([
-        url(r'^$', TemplateView.as_view(template_name='events/index.html'),
-            name='events'),
-        url(r'^(?P<doc_id>[\w-]+)/$',
-            SheerTemplateView.as_view(doc_type='events',
-                                      local_name='event',
-                                      default_template='events/_single.html'),
-            name='event'),
-        url(r'^(?P<doc_id>[\w-]+)/ics/$', EventICSView.as_view())],
-        namespace='events')),
-
```

Comment:
yea this was for testing purposes to see the page renders correctly and since i was creating events under the events landing page, the `/events/` slug was interfering. i can revert this in this PR since the molecules haven't been assigned to specific pages yet.